README.md

```diff
@@ -46,18 +48,30 @@ Sends requests to the server, which replies with responses.
 
 #### Server
 
-Serves clients over websockets.
 Continuously listens to TTYs.
+Serves clients over websockets, with the messages received via the TTYs.
 
 #### Endpoint
 
-A thing which may produce messages and accepts being written to.
+A thing which may produce serial messages and accepts being written to.
 
 For example the endpoint `/dev/ttyACM0` or `COM0` represents real TTYs which can produce messages.
 
 Endpoints may also be mocked, and thus have any name e.g. `mock-1` or `specific-mocked-device-30`.
 
-### Example: Client TTY session
+## Cargo Features
+
+### `mocks-share-endpoints`
```

Comment:
Moved this before the examples, else you end up in the mocking example without even heard of the feature yet, making it kind of a surprise.